go/libkb/kbsig.go

```diff
@@ -465,6 +467,45 @@ func SignJSON(jw *jsonw.Wrapper, key GenericKey) (out string, id keybase1.SigID,
 	return
 }
 
+func GetDefaultSigVersion(g *GlobalContext) SigVersion {
+	if g.Env.IsAdmin() {
```

Comment:
So this will be flagged on for most of us in the office? If we're worried about it breaking something, is that restricted enough?

Reply:
I'll defer to @maxtaco or @chrisnojima -- is there a more restrictive flag we can use? I think given the nature of the bugs that can be produced it will be hard to absolutely guarantee it doesn't affect others.

Reply:
@patrickxb  how have we been feature-flagging on Go?

Reply:
That IsAdmin() is useless...we should just remove it.

`G.Env.GetFeatureFlags()` is preferable.

Reply:
removed and fixed

Reply:
removed and fixed